src/bp/ui-studio/src/web/components/Layout/index.tsx

```diff
@@ -64,9 +61,26 @@ const Layout: FC<ILayoutProps> = props => {
 
     setTimeout(() => BotUmountedWarning(), 500)
 
+    const handleWebChatPanel = message => {
+      if (message.data.name === 'webchatOpened') {
+        props.emulatorOpen(true)
```

Comment:
I'd name this "setEmulatorOpen" , otherwise we may think it's a boolean indicating the state

Reply:
yes